speechmatics/client.py

```diff
@@ -519,6 +519,33 @@ def run_synchronously(self, *args, timeout=None, **kwargs):
         # pylint: disable=no-value-for-parameter
         asyncio.run(asyncio.wait_for(self.run(*args, **kwargs), timeout=timeout))
 
+    async def send_message(self, msg):
```

Comment:
why is this this wrapper function necessary and not call `self.client.websocket.send()` directly?

Reply:
It adds a bit of error checking and helpful messages back to a user + it asserts that websocket is not None and that a session is actually running. Thought it would be nicer to have an explicit client method for that? Otherwise in any user code, to make sure it is type-checked you would have to do:

```python
if client.websocket is not None:
     client.websocket.send(message)
```